check_dvcs.py

```diff
@@ -60,10 +58,10 @@ def delete_previous_comments(comments_urls: list[str]) -> None:
         raise CommandException('\n'.join(comments_that_failed_to_delete))
 
 
-def does_string_start_with_jira(string_to_match: str) -> Optional[str]:
-    pr_title_re = re.compile(f"^({_AAP_RE}|{_NO_JIRA_MARKER})", re.IGNORECASE)
+def does_string_contain_jira(string_to_match: str) -> Optional[str]:
+    pr_title_re = re.compile(f"({_AAP_RE}|{_NO_JIRA_MARKER})")
     matches = pr_title_re.match(string_to_match)
```

Comment:
I think `.match()` is only used for finding pattern at the start of the string. For strings like "a-hoopy-AAP-xxxxxx-frood", `.match()` returns None. This is contradicting to our acceptance criteria.

Reply:
Yeah, this was a good catch, we should `.search()` instead of `.match()`